update_chroot

```diff
@@ -58,7 +66,7 @@ sudo touch "/etc/portage/make.conf.user"
 sudo_clobber "/etc/portage/make.conf" <<EOF
 # Created by update_chroot
 PORTDIR="${PORTAGE_STABLE_OVERLAY}"
-PORTDIR_OVERLAY="${CROSSDEV_OVERLAY} ${COREOS_OVERLAY}"
+PORTDIR_OVERLAY="${CROSSDEV_OVERLAY} ${COREOS_OVERLAY} ${EXTRA_PORTDIR_OVERLAY}"
```

Comment:
The PORTDIR* variables are actually deprecated and at some point I should prune references to them. The newer scheme is to add configs under repos.conf. Does it work to just write out /etc/portage/repos.conf/local.conf and not bother with adding it to the variable? A special config file would have the added benefit preserving this setting when update_chroot gets called later, likely without the new overlay option. As is this setting will get lost as soon as you run build_packages.